src/debug.cpp

```diff
@@ -136,7 +140,9 @@ static void debug_print(const char *dbg, const char *buf)
 		char buffer[512];
 		seprintf(buffer, lastof(buffer), "%sdbg: [%s] %s\n", GetLogPrefix(), dbg, buf);
 #if defined(WIN32) || defined(WIN64)
-		_fputts(OTTD2FS(buffer, true), stderr);
+		TCHAR system_buf[512];
+		convert_to_fs(buffer, system_buf, lengthof(system_buf), true);
+		_fputts(system_buf, stderr);
 #else
```

Comment:
What does this change do?

Reply:
It makes sure that if you call DEBUG from a thread, it doesn't trash the global string buffer that is used by OTTD2FS.